packages/docs/docs/docs/model/01-definition.md

```diff
@@ -1,284 +1,382 @@
 # Definition
 
-The `@mondrian-framework/model` package contains a wide range of useful functions for defining a data model schema, from the simplest type `string` to complex objects, arrays, and unions. The syntax used has been designed to make development straightforward and the schema as readable as possible.
+The `@mondrian-framework/model` package contains a wide range of useful functions for defining a data model schema, from primitive types to complex objects, arrays, and unions. The syntax used has been designed to make development straightforward and the schema as readable as possible.
+
+Everything you may need to define a new schema is defined inside the `types` namespace of the `@mondrian-framework/model` package, so to get things started you should import it:
+
+```ts
+import { types } from "@mondrian-framework/model"
+```
 
 ## Primitives
-Mondrian Framework supports the definition of a really small, simple but powerfull range of primitive types.
 
-```ts showLineNumbers
```

Comment:
Did you remove line numbers intentionally?

Reply:
I thought it would be useful to keep them only for complex examples with multiple pieces so I removed it from simple code snippets. If you want I can add it back :)